src/commands/generate/app.ts

```diff
@@ -122,17 +122,44 @@ export default class GenerateApp extends BaseCommand<typeof GenerateApp> {
           },
           name: `${config.name} [id: ${config.configurationId}]`,
         }))
+        choices.push({
+          value: {
+            id: 'new-config',
+            auth: undefined,
+          },
+          name: 'Create new login config',
+        })
         const selectedConfig = await select({
           message: 'Select a login configuration to use in your reference application',
           choices,
         })
-        const clientSecret = validateInputLength(
-          await password({
-            message: "What is the login configuration's client secret?",
-            mask: true,
-          }),
-          INPUT_LIMIT,
-        )
+        let newConfigClientSecret = undefined
+        // Create a new login config
+        if (selectedConfig.id === 'new-config') {
+          const newConfigName = validateInputLength(
+            await input({ message: `Enter a name for the login config` }),
+            INPUT_LIMIT,
+          )
+          const redirectUri = getRedirectUri(GenerateApp.apps, appName)
+          const createLoginConfigInput = {
+            name: newConfigName,
+            redirectUris: [redirectUri],
+          }
+          const createConfigOutput = await vpAdapterService.createLoginConfig(createLoginConfigInput)
```

Comment:
Are we properly handling errors from the creation? Did you try creating a login config with invalid name?

Reply:
It's handled by the vpAdapterService [here](https://github.com/affinidi/affinidi-cli/blob/main/src/services/affinidi/vp-adapter/index.ts#L52).
a screenshot for blank name
<img width="785" alt="Screenshot 2024-01-04 at 5 03 21 PM" src="https://github.com/affinidi/affinidi-cli/assets/146182581/74a0d5bd-29a3-40c5-abdf-b9ffc0c04027">